README.md

```diff
@@ -1,5 +1,8 @@
 # fluent-plugin-s3-arrow
 
+[![Gem Version](https://badge.fury.io/rb/fluent-plugin-s3-arrow.svg)](https://badge.fury.io/rb/fluent-plugin-s3-arrow)
+[![Docs](https://img.shields.io/badge/docs-latest-blue.svg)](https://rubydoc.info/gems/fluent-plugin-s3-arrow)
```

Comment:
```suggestion
```

This plugin is not a library, so it's better not to link to rdoc.

Reply:
Thanks done!